roles/haproxy-config/templates/lb_backend.j2

```diff
@@ -0,0 +1,33 @@
+
+{% for be in lb_entries %}
+
+{% if ((be.lb_ssl_enabled is defined and be.lb_ssl_enabled) or
+       (be.lb_ssl_enabled is undefined and be.lb_port|string == '443')) %}
+{% set lb_ssl_enabled = True %}
```

Comment:
@oybed there should be an else clause to ensure the variable is always set based on the conditional

```
{% if ((be.lb_ssl_enabled is defined and be.lb_ssl_enabled) or
       (be.lb_ssl_enabled is undefined and be.lb_port|string == '443')) %}
{% set lb_ssl_enabled = True %}
{% else %}
{% set lb_ssl_enabled = False %}
{% endif %}
```